install_ddev.sh

```diff
@@ -0,0 +1,197 @@
+#!/bin/bash
```

Comment:
This is one of many ways to install DDEV. That's not the itention of this repository. Just remove it, please.